source/installation.txt

```diff
@@ -0,0 +1,81 @@
+.. _scala-install:
+
+============
+Installation
+============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: dependency, reactive streams, add package
+
+The recommended way to get started with the {+driver-short+} in your
+project is by using a dependency management system.
+
+The following sections contain the Reactive Streams-based Scala
+implementation for asynchronous stream processing with non-blocking back
+pressure.
+
+Scala 2.12 Based Driver
+-----------------------
+
+.. tabs::
+
+   .. tab:: Maven
+      :tabid: maven-installation
+
+      If you are using `Maven <https://maven.apache.org/>`__ to manage your
+      packages, add the following entry to your ``pom.xml`` dependencies list:
+
+      .. code-block:: xml
+
+         <dependencies>
+             <dependency>
+                 <groupId>org.mongodb.scala</groupId>
+                 <artifactId>mongo-scala-driver</artifactId>
+                 <version>5.0.0_2.12</version>
```

Comment:
Suggestion:
Since the version uses the "driver version"_"Scala version" naming scheme, I think it could be helpful to use a source constant for the "driver version" portion.

Applies to both instances.